hardware/Kodi.h

```diff
@@ -9,8 +9,9 @@
 #include <boost/asio.hpp>
 #include <boost/array.hpp>
 #include <boost/enable_shared_from_this.hpp>
+#include <boost/lexical_cast.hpp>
```

Comment:
I'm not a fan of depending on lexical cast, cant we use the stringstream ? what is wrong with this